civictechprojects/static/css/partials/_PositionList.scss

```diff
@@ -0,0 +1,51 @@
+.PositionList-entry {
+  background-color: $color-background-light;
+  box-shadow: 0 3px 5px rgba(0,0,0,0.2);
+  padding: 5px;
+  margin: 15px;
+  max-width: 600px;
+  display: flex;
+  justify-content: space-between;
```

Comment:
Two things here:
 - Add `align-items: center;` so the role title text ("Business: Analyst") is also vertically centered
 - if the design doc specifies a minimum height, I'd add that too. If not, I'd ...probably still add something like `min-height: 40px;` because these are touch targets on mobile and need to be big enough.